toolbox/_common.sh

```diff
@@ -0,0 +1,11 @@
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
WORK_DIR, TESTING_DIR, I can think on a few names a bit better than THIS, brings me bad memories from javascript

Reply:
hum, my first answer is:

I prefer not to change this one, as `THIS_DIR` will point to different location depending where you your file lives, so WORK_DIR/TESTING_DIR don't exactly match

BUT

here, there's a conflict between `_common.sh` file and the script that will source it,
so I'll change it here to `TOOLBOX_DIR`, but as per the answer above, I prefer keeping it in the other scripts

Reply:
(and see [google search](https://www.google.com/search?client=firefox-b-d&ei=-qobYKyyH6ielwTkta_gCg&q=%22THIS_DIR%22+bash&oq=%22THIS_DIR%22+bash&gs_lcp=CgZwc3ktYWIQAzIGCAAQFhAeOgQIABANOgQILhANOgQIABAeOgYIABAHEB5QjRRY0yRg2yloAHAAeACAAZcBiAHaB5IBAzAuOJgBAKABAaoBB2d3cy13aXrAAQE&sclient=psy-ab&ved=0ahUKEwisl7Tt4s_uAhUoz4UKHeTaC6wQ4dUDCAw&uact=5), it's a common way to name this variable)